### PR TITLE
Update readme for outputPaths

### PR DESCRIPTION
**Pull request by TadCordle**

### CODE REVIEW

File: jib-maven-plugin/README.md [hunk 1]
```diff
@@ -210,11 +209,6 @@ Then, you can build your container image by running:
 mvn package
 ```
 
-### Additional Build Artifacts
-
-As part of an image build, Jib also writes out the _image digest_ to
```
Comment (chanseokoh) [inline jib-maven-plugin/README.md]: Isn't this the default behavior that didn't change? If so, it's fine to leave this, just adding "... by default" and "the names (paths) are configurable?" You thought it's just good to remove this?

Comment (TadCordle) [reply jib-maven-plugin/README.md]: It seemed a bit redundant after adding it to the "Extended Usage" section, considering those list the defaults.

Comment (chanseokoh) [reply jib-maven-plugin/README.md]: I think it's meaningful to call out the additional artifacts here. The "Extended Usage" is like an API reference that people don't look into it until necessary.

File: jib-gradle-plugin/README.md [hunk 2]
```diff
@@ -24,7 +24,6 @@ For information about the project, see the [Jib project README](../README.md).
     * [Build to Docker Daemon](#build-to-docker-daemon)
     * [Build an image tarball](#build-an-image-tarball)
   * [Run `jib` with each build](#run-jib-with-each-build)
-  * [Additional Build Artifacts](#additional-build-artifacts)
```
Comment (chanseokoh) [inline jib-gradle-plugin/README.md]: Sorry, missed this one. This can be restored.

File: jib-maven-plugin/README.md [hunk 3]
```diff
@@ -24,7 +24,6 @@ For information about the project, see the [Jib project README](../README.md).
     * [Build to Docker Daemon](#build-to-docker-daemon)
     * [Build an image tarball](#build-an-image-tarball)
   * [Bind to a lifecycle](#bind-to-a-lifecycle)
-  * [Additional Build Artifacts](#additional-build-artifacts)
```
Comment (chanseokoh) [inline jib-maven-plugin/README.md]: ditto

File: jib-maven-plugin/README.md [hunk 4]
```diff
@@ -281,6 +280,14 @@ Property | Type | Default | Description
 `paths` | list | `[(project-dir)/src/main/jib]` | List of extra directories. Can be absolute or relative to the project root.
 `permissions` | list | *None* | Maps file paths on container to Unix permissions. (Effective only for files added from extra directories.) If not configured, permissions default to "755" for directories and "644" for files.
 
+<a name="outputpaths-object"></a>`outputPaths` is an object with the following properties:
+
+Property | Type | Default | Description
+--- | --- | --- | ---
+`tar` | string | `(project-dir)/target/jib-image.tar` | The path of the tarball generated by `jib:buildTar`. Relative paths are resolved relative to the project root.
+`digest` | string | `(project-dir)/target/jib-image.digest` | The path of the image digest written out during the build. Relative paths are resolved relative to the project root.
+`imageId` | string | `(project-dir)/target/jib-image.id` | The path of the image id written out during the build. Relative paths are resolved relative to the project root.
```
Comment (chanseokoh) [inline jib-maven-plugin/README.md]: nit: image *ID*

File: jib-gradle-plugin/README.md [hunk 5]
```diff
@@ -228,13 +227,21 @@ Property | Type | Default | Description
...
 `permissions` | `Map<String, String>` | *None* | Maps file paths on container to Unix permissions. (Effective only for files added from extra directories.) If not configured, permissions default to "755" for directories and "644" for files.
 
+<a name="outputpaths-closure"></a>`outputPaths` is a closure with the following properties:
+
+Property | Type | Default | Description
+--- | --- | --- | ---
+`tar` | `File` | `(project-dir)/build/jib-image.tar` | The path of the tarball generated by `jib:buildTar`. Relative paths are resolved relative to the project root.
+`digest` | `File` | `(project-dir)/build/jib-image.digest` | The path of the image digest written out during the build. Relative paths are resolved relative to the project root.
+`imageId` | `File` | `(project-dir)/build/jib-image.id` | The path of the image id written out during the build. Relative paths are resolved relative to the project root.
```
Comment (chanseokoh) [inline jib-gradle-plugin/README.md]: nit: image *ID*

**Pull request merged**